common/features/config/networks/static/reducer.ts

```diff
@@ -850,6 +851,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       max: 60,
       initial: 1
     }
+  },
+  WEB: {
+    id: 'WEB',
+    name: 'Webchain',
+    unit: 'WEB',
+    chainId: 24484,
+    isCustom: false,
+    color: '#0092ee',
+    blockExplorer: makeExplorer({
+      name: 'Webchain Explorer',
+      origin: 'https://explorer.webchain.network'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: WEB_DEFAULT,
```

Comment:
Please include links to the relevant pull request in third party repositories. It saves time for those reviewing the PR. 

No pull request currently exists for Trezor support. Please remove TREZOR support from this PR until after you can demonstrate Ledger hardware wallet support.